_layouts/base.html

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="{{ page.lang | default: 'en-US' }}">
-
+  <script language="javascript" type="text/javascript">
+    window._locale_data = {{site.data[page.lang].settings.po_search | jsonify}}
+  </script>
```

Comment:
this adds the locale data to all pages? I would see if we can have a way to set this for only the one PO search page

Reply:
some ideas:

```
{% if page.locale_data %}
  < .... >
{% endif %}
```

and that would mean the translations live in each page's YAML frontmatter

```
lang: en
# ...
locale_data:
  some_string: 'aaa'
```